src/views/count.vue

```diff
@@ -55,7 +55,7 @@
           </ion-item>
           <ion-item>
             <ion-label>{{ $t("Location") }}</ion-label>
-            <ion-chip @click="selectLocation">
+            <ion-chip @click="selectLocation" class="remove-count">
```

Comment:
This chip has an actions associated with it, so, we shouldn't remove the pointer mouse from here.